<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListWidget" name="descargaslistWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>91</y>
     <width>256</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>149</y>
     <width>160</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="descargarButton">
      <property name="text">
       <string>Descargar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="rechazarButton">
      <property name="text">
       <string>Rechazar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="volverButton">
      <property name="text">
       <string>Volver</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>341</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Alguien te ha enviado un archivo</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
